src/Kerbero.WebApi/appsettings.Development.json

```
@@ -15,6 +15,6 @@
     "BaseUrl": "https://api.nuki.io"
   },
   "ConnectionStrings": {
-    "DevDatabases" : "Host=my_host;Database=my_db;Username=my_user;Password=my_pw"
+    "DevDatabase" : "Host=localhost:5432;Database=kerberodb;Username=kuama;Password=kuama"
```

```suggestion
    "Database" : "Host=localhost:5432;Database=kerberodb;Username=kuama;Password=kuama"
```
we're already inside `appsettings.Development.json`